### PR TITLE
bug-1900646: fix submitter to only consume standard queue

### PR DESCRIPTION
**Pull request by willkg**
This fixes the submitter to only consume from the standard subscription rather than from standard, priority, and reprocessing.

It does it by stomping on the subscription_name settings for priority and reprocessing in the PubSubCrashQueue class. That means this will never run in AWS, but that's what our plan was anyhow.

### CODE REVIEW

File: socorro/external/pubsub/crashqueue.py [hunk 1]
```diff
@@ -244,7 +261,11 @@ def publish(self, queue, crash_ids):
                 try:
                     future.result()
                 except Exception:
-                    logger.exception(f"Crashid failed to publish: {batch[i]}")
+                    logger.exception(
```
Comment (willkg) [inline socorro/external/pubsub/crashqueue.py:264]: This is an unrelated fix to a bad exception logging line I did earlier. Boo me.

**Review by relud — APPROVED** [review]
lgtm, but after reading i realize we could forgo this complexity and instead specify the same subscription for every queue in helm. i'm happy either way.
